asset/README.md

```diff
@@ -0,0 +1,17 @@
+<!-- THIS FILE IS AUTO-GENERATED, EDIT docs/asset/overview.md -->
```

Comment:
Is this comment right?  Is there a specific workflow we need to follow to update this?